Makefile

```diff
@@ -51,7 +51,9 @@ tag-client-js: test-client-js
 
 .PHONY: test-client-js
 test-client-js: build-client-js
-	make run-in-docker sdk_language=js image=node:${NODE_DOCKER_TAG} command="/bin/sh -c 'npm test ; npm audit ; npm run lint'"
+	make run-in-docker sdk_language=js image=node:${NODE_DOCKER_TAG} command="/bin/sh -c 'npm test'"
```

Comment:
Can you use `&&` so they're all still in one command?
`npm test && npm audit && npm run lint`

Also good catch! Let's do it to all the sdks that have the same mistake

Reply:
The other SDKs don't have the same mistake as they are broken up in multiple lines.